pyflakes/checker.py

```diff
@@ -799,7 +799,7 @@ def on_conditional_branch():
             return
 
         if on_conditional_branch():
-            # We can not predict if this conditional branch is going to
+            # We cannot predict if this conditional branch is going to
```

Comment:
Seems like "cannot" is [much more common](http://www.oxforddictionaries.com/words/cannot-or-can-not) though.